### PR TITLE
camel case handling

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

File: .spellcheck.yml [hunk 1]
```diff
@@ -59,5 +65,5 @@ matrix:
       context_visible_first: true
       delimiters:
       # Ignore multiline content between [source, *] and blank line
-      - open: '\[source,.*\]'
-        close: '^\s*$' 
+      - open: '\[source, .*\]\r?\n\s*----'
+        close: '[^-]*'
```
Comment (ssarmokadam) [inline .spellcheck.yml:69]: Change it to latest regex. 
```
 - pyspelling.filters.context:
      context_visible_first: true
      delimiters:
      # Ignore multiline content between [source, *] and blank line
      - open: '\[source,.*\]\r?\n\s*----'
        close: '\s*----'
```

File: documentation/build-devon4j-application.asciidoc [hunk 2]
```diff
@@ -15,11 +15,11 @@ In this step we are going to create our work environment and establish the first
 
 == Setting up a devonfw Work Environment
 Follow the setup instructions listed https://github.com/devonfw/devon-ide/blob/master/documentation/setup.asciidoc[here]. +
-After you have done so, your working directroy should look like this:
+After you have done so, your working directory should look like this:
 
 image::images/devon4j/3.BuildYourOwn/after_setup.png[After Setup, 250]
 
-Press "Shift + Right Click" and select "Open Devon CMD shell here". +
+Press "Shift + Right Click" and select "Open Devon `CMD` shell here". +
```
Comment (ssarmokadam) [inline documentation/build-devon4j-application.asciidoc:22]: cmd should not be in backtick... it is in dict check and remove backtick

File: documentation/build-devon4j-application.asciidoc [hunk 3]
```diff
@@ -56,7 +56,7 @@ We are using the `java` https://github.com/devonfw/devon-ide/blob/master/documen
 ====
 [WARNING]
 ====
-Be sure that you are using devonfw CMD or bash, as the command is not working properly with devonfw PowerShell.
+Be sure that you are using devonfw `CMD` or bash, as the command is not working properly with devonfw `PowerShell`.
```
Comment (ssarmokadam) [inline documentation/build-devon4j-application.asciidoc:59]: cmd and powershell in dict

File: documentation/build-devon4j-application.asciidoc [hunk 4]
```diff
@@ -151,7 +151,7 @@ You are redirected to a login screen because -- by default -- a new _devon4j_ ap
 When creating _devon4j_ based apps this way, we get the following features _out-of-the-box_:
 
 * _Maven_ project with _api_ project, _core_ project and _server_ project:
-** _api_ project for the common API
+** _api_ project for the common `API`
```
Comment (ssarmokadam) [inline documentation/build-devon4j-application.asciidoc:154]: api -> dict

File: documentation/devon4j-adding-custom-functionality.asciidoc [hunk 5]
```diff
@@ -127,9 +127,9 @@ public class QueuemanagementImpl extends AbstractComponentFacade implements Queu
 
 These methods are simply going to call the `ucManageQueue` methods that were just added. 
 
-=== Creating the usecasemanage for the AccessCode
+=== Creating the usecasemanage for the `AccessCode`
```
Comment (ssarmokadam) [inline documentation/devon4j-adding-custom-functionality.asciidoc:130]: remove backtick from heading and divide word as Access Code

File: documentation/devon4j-adding-custom-functionality.asciidoc [hunk 6]
```diff
@@ -127,9 +127,9 @@ public class QueuemanagementImpl extends AbstractComponentFacade implements Queu
...
 
-==== Adding method to the AccessCode _usecasefind_
+==== Adding method to the `AccessCode` _usecasefind_
```
Comment (ssarmokadam) [inline documentation/devon4j-adding-custom-functionality.asciidoc:132]: same as above

File: documentation/devon4j-adding-custom-functionality.asciidoc [hunk 7]
```diff
@@ -454,7 +454,7 @@ public class AccesscodemanagementImpl extends AbstractComponentFacade implements
 ----
 
 === Adding to the Service
-To add the new service, we need to add its definition to the `jtqj-api` in `accesscodemanagement/service/api/rest/AccesscodemanagementRestService.java`. We are going to create a new `/accessCode` REST resource bound to three methods, one called _findAccessCodeEtos_, another one called _saveAccessCode_ and yet another one called _deleteAccessCode_.
+To add the new service, we need to add its definition to the `jtqj-api` in `accesscodemanagement/service/api/rest/AccesscodemanagementRestService.java`. We are going to create a new `/accessCode` `REST` resource bound to three methods, one called _findAccessCodeEtos_, another one called _saveAccessCode_ and yet another one called _deleteAccessCode_.
```
Comment (ssarmokadam) [inline documentation/devon4j-adding-custom-functionality.asciidoc:457]: REST-> dict

File: documentation/devon4j-adding-custom-functionality.asciidoc [hunk 8]
```diff
@@ -531,9 +531,9 @@ public class AccesscodemanagementRestServiceImpl implements Accesscodemanagement
 Now run the app again via Eclipse and use Postman to call the new save service via *POST*: +
 `http://localhost:8081/jumpthequeue/services/rest/accesscodemanagement/v1/accesscode/`
 
-In the body, provide an AccessCode object with the following required parameters:
+In the body, provide an `AccessCode` object with the following required parameters:
 
-//TODO: This fails due to an SQL Error: 23506
+//TODO: This fails due to an `SQL` Error: 23506
```
Comment (ssarmokadam) [inline documentation/devon4j-adding-custom-functionality.asciidoc:536]: SQL->dict

File: documentation/devon4j-adding-custom-functionality.asciidoc [hunk 9]
```diff
@@ -545,13 +545,13 @@ In the body, provide an AccessCode object with the following required parameters
...
 
-image::images/devon4j/6.Customizations/jumpthequeue_accesscode.png[JumpTheQueue AccessCode]
+image::images/devon4j/6.Customizations/jumpthequeue_accesscode.png[JumpTheQueue `AccessCode`]
 
 In order to know, if the new codeaccess has been successfully created, we can search all the CTOs, like we did in the previous steps. The new accesscode should be on the bottom:
 
 image::images/devon4j/6.Customizations/jumpthequeue_listwithcode.png[JumpTheQueue List with Code]
 
-To test the delete, you can send a *DELETE* to this URL:
+To test the delete, you can send a *DELETE* to this `URL`:
```
Comment (ssarmokadam) [inline documentation/devon4j-adding-custom-functionality.asciidoc:554]: URL->dict

File: documentation/devon4j-components.asciidoc [hunk 10]
```diff
@@ -35,7 +35,7 @@ A component consists of three packages, which correspond to the three layers def
 
 image::images/devon4j/4.Components/layers.png[Component Layers, 300]
 
-* _Service_ layer: exposes the REST API to exchange information with client applications
+* _Service_ layer: exposes the `REST` API to exchange information with client applications
```
Comment (ssarmokadam) [inline documentation/devon4j-components.asciidoc:38]: REST->dict

File: documentation/devon4j-components.asciidoc [hunk 11]
```diff
@@ -57,7 +57,7 @@ The naming convention for these entities in _devon4j_ is:
 
 Basically, each _Entity_ is a https://en.wikipedia.org/wiki/Plain_old_Java_object[POJO] (plain old Java object) that will be mapped to a table in the database and represent each column via a suitable property.
 
-.An entity and its corresponding table in the DB (taken from another application).
+.An entity and its corresponding table in the `DB` (taken from another application).
```
Comment (ssarmokadam) [inline documentation/devon4j-components.asciidoc:60]: DB to dict

File: documentation/devon4j-components.asciidoc [hunk 12]
```diff
@@ -239,7 +239,7 @@ CREATE TABLE AccessCode(
 * _idQueue_: The relation with the _DailyQueue_ table.
 
 ==== Mock Data
-Finally we are going to provide a certain amount of mock data, which will be available right from the start in our application. Create a new SQL script in `/jtqj-core/src/main/resources/db/migration/1.0/`, called:
+Finally we are going to provide a certain amount of mock data, which will be available right from the start in our application. Create a new `SQL` script in `/jtqj-core/src/main/resources/db/migration/1.0/`, called:
```
Comment (ssarmokadam) [inline documentation/devon4j-components.asciidoc:242]: SQL->dict

File: documentation/devon4j-components.asciidoc [hunk 13]
```diff
@@ -274,7 +274,7 @@ INSERT INTO AccessCode (id, modificationCounter, ticketNumber, creationTime, sta
 === The Core of the Components
 Now that we have defined the database for our entities, we should start creating the code of the related components.
 
-We are going to use CobiGen to generate the component structure. That means that -- as already commented -- we can generate all the structure and layers starting from a _core_ element: a simple _Plain Old Java Object_ that represents our _Entity_. So, in order to use CobiGen, we have to create our entities in the expected locations (as you will see in the following section): `<entitymanagement>.dataaccess.api`.
+We are going to use `CobiGen` to generate the component structure. That means that -- as already commented -- we can generate all the structure and layers starting from a _core_ element: a simple _Plain Old Java Object_ that represents our _Entity_. So, in order to use `CobiGen`, we have to create our entities in the expected locations (as you will see in the following section): `<entitymanagement>.dataaccess.api`.
```
Comment (ssarmokadam) [inline documentation/devon4j-components.asciidoc:277]: CobiGen->dict

File: documentation/devon4j-components.asciidoc [hunk 14]
```diff
@@ -623,7 +623,7 @@ We will end up with the following structure:
 
 image::images/devon4j/4.Components/queue_entity.png[Queue Entity, 250]
 
-The contents of _QueueEntity_ before using CobiGen will be:
+The contents of _QueueEntity_ before using `CobiGen` will be:
```
Comment (ssarmokadam) [inline documentation/devon4j-components.asciidoc:626]: CobiGen->dict

File: documentation/devon4j-components.asciidoc [hunk 15]
```diff
@@ -761,22 +761,22 @@ Now we have finished preparing the _core_ of our components and can start using
 
 [IMPORTANT]
 ====
-Now we can resolve the compilation errors related to `QueueEntity` in the `AccessCodeEntity.java` by applying the suggestions of the IDE. To do this, open the offending file, click the first red light bulb on the left border of the editor and select `Import 'QueueEntity' (com.devonfw. ...)`.
+Now we can resolve the compilation errors related to `QueueEntity` in the `AccessCodeEntity.java` by applying the suggestions of the `IDE`. To do this, open the offending file, click the first red light bulb on the left border of the editor and select `Import 'QueueEntity' (com.devonfw. ...)`.
```
Comment (ssarmokadam) [inline documentation/devon4j-components.asciidoc:764]: IDE->dict

File: documentation/devon4j-components.asciidoc [hunk 16]
```diff
@@ -761,22 +761,22 @@ Now we have finished preparing the _core_ of our components and can start using
...
 
 Or just manually add this line to your import statements:
 ----
 import com.devonfw.application.jtqj.queuemanagement.dataaccess.api.QueueEntity;
 ----
 ====
 
-=== The Component Structure (using CobiGen)
-Once we are finished creating the _core_ of our components we could continue to create the structure and all elements manually, but we are going to use CobiGen for these tasks, since we can save a significant amount of time and effort this way.
+=== The Component Structure (using `CobiGen`)
```
Comment (ssarmokadam) [inline documentation/devon4j-components.asciidoc:772]: same as above

File: documentation/devon4j-components.asciidoc [hunk 17]
```diff
@@ -785,72 +785,72 @@ devon eclipse add-plugin cobigen
...
 
-For now just click `Finish` and let CobiGen do its work.
+For now just click `Finish` and let `CobiGen` do its work.
 
 [NOTE]
 ====
 In detail the selected options do the following:
 
-* _CRUD SpringData Repository_: Generates the entity repository (that contains the CRUD operations) in the data access layer.
+* _CRUD `SpringData` Repository_: Generates the entity repository (that contains the `CRUD` operations) in the data access layer.
```
Comment (ssarmokadam) [inline documentation/devon4j-components.asciidoc:828]: CRUD->dict

File: documentation/devon4j-components.asciidoc [hunk 18]
```diff
@@ -866,14 +866,14 @@ First, we need to adjust manually some imports related to _Timestamp_ in:
 
 We can fix these errors manually by adding `*import* java.sql.Timestamp` to the affected Java files:
 
-image::images/devon4j/4.Components/cobigen5_manual_import.png[CobiGen Manual Import]
+image::images/devon4j/4.Components/cobigen5_manual_import.png[`CobiGen` Manual Import]
```
Comment (ssarmokadam) [inline documentation/devon4j-components.asciidoc:869]: change all above like previous comments

File: documentation/devonfw-intro.asciidoc [hunk 19]
```diff
@@ -26,7 +26,7 @@ Having all the dependencies serf-contained in the distribution's _zip_ file, use
 == Pre-packaged Software
 The list of software pre-installed with devonfw distributions is:
 
-- _Eclipse_: pre-configured and fully functional IDE to develop Java based apps.
+- _Eclipse_: pre-configured and fully functional `IDE` to develop Java based apps.
```
Comment (ssarmokadam) [inline documentation/devonfw-intro.asciidoc:29]: same as above

File: documentation/devonfw-intro.asciidoc [hunk 20]
```diff
@@ -65,9 +65,9 @@ All those tasks can be done manually using _Maven_, _Tomcat_, _Sencha Cmd_, _Bow
 
 === Cobigen
 
-_Cobigen_ is a code generator included in the context of _devonfw_ that allows users to generate all the structure and code of the components, helping to save a lot of time waisted in repetitive tasks.
+_Cobigen_ is a code generator included in the context of _devonfw_ that allows users to generate all the structure and code of the components, helping to save a lot of time consumed in repetitive tasks.
 
-image::images/devon/cobigen.png[CobiGen]
+image::images/devon/cobigen.png[`CobiGen`]
```
Comment (ssarmokadam) [inline documentation/devonfw-intro.asciidoc:70]: remove backticks

File: documentation/devonfw-intro.asciidoc [hunk 21]
```diff
@@ -90,4 +90,4 @@ The current available modules are:
 - _winauth single sign on_: module that allows applications to authenticate the users by the Windows credentials.
 
 '''
-*Next Chapter*: link:devonfw-ide-setup.asciidoc[devonfw IDE Setup]
\ No newline at end of file
+*Next Chapter*: link:devonfw-ide-setup.asciidoc[devonfw `IDE` Setup]
```
Comment (ssarmokadam) [inline documentation/devonfw-intro.asciidoc:93]: remove backticks

**Review by ssarmokadam — CHANGES_REQUESTED** [review]

File: .spellcheck.yml (hunk 1 above)
Comment (dixyushi) [reply .spellcheck.yml]: Updated the code

**Comment by ssarmokadam** [comment]
@suprishi will take over this PR. He will work on review comments and raise new PR. Merging this basic changes.

**Pull request merged**